runtime/Stdlib_Error.resi

```diff
@@ -149,34 +149,15 @@ Raises (throws in JavaScript language) the provided `Error.t`, which will stop e
 let error = Error.make("Everything is upside down.")
 
 if 5 > 10 {
-  error->Error.raise
+  throw(error->Error.toException)
 } else {
   Console.log("Phew, sanity still rules.")
 }
 ```
 */
-@deprecated(
-  "`raise` has been renamed to `throw` to align with JavaScript vocabulary. Please use `throw` instead"
-)
+@deprecated("Please use the global `throw` or `throwAny` function instead")
```

Comment:
I don't understand, these two are not the same, `raise` here is meant to be used with a JsException, I think there's still a use case for it. Deprecating it in favor of `Error.throw` or `Error.throwJsException` or something similar would still make sense to me.

Reply:
@tsnobip I meant that for a JS error you could either use

```rescript
throw(error->Error.toException)
```

or

```
```rescript
throwAny(error)
```

and so one does not need a separate `Error.throw` function.